README.md

```diff
@@ -1,5 +1,5 @@
 # Nessie-iOS-Wrapper
-### Note: This SDK is Swift 3.0 thanks to [UberJason](https://github.com/UberJason). Please use Xcode 8 and later 😀
+### Note: This SDK is Swift 5.0, please use Xcode 8 and later 😀
```

Comment:
Something small but can you say Xcode 10 and later here?

Reply:
Fixed, thanks!